.github/workflows/benchmark.yml

```
@@ -0,0 +1,38 @@
+name: Benchmarking
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*" 
```

maybe add also a manual button for testing and redo a test?

```suggestion
  workflow_dispatch:
  push:
    branches:
      - master
    tags:
      - "v*" 
```

---

The manual button for testing is [here](https://github.com/jina-ai/jina-terraform/actions/workflows/benchmark.yml). This workflow mainly dispatches this trigger to [jina-ai/jina-terraform](https://github.com/jina-ai/jina-terraform) repo.

I will create a redo button there as well.